pkg/instrumentation/sdk.go

```diff
@@ -208,6 +208,10 @@ func (i *sdkInjector) injectCommonSDKConfig(ctx context.Context, otelinst v1alph
 }
 
 func chooseServiceName(pod corev1.Pod, resources map[string]string, index int) string {
+	serviceNames := strings.Split(pod.Annotations["instrumentation.opentelemetry.io/service-name"], ",")
```

Comment:
extract the annotation to a constant

Reply:
I did the split in the case of people with multiple containers. for you to be able to override the different container names